KernelStubs.php

```diff
@@ -1091,7 +1091,20 @@ public static function setVariableProfileAssociation(string $ProfileName, float
             self::checkVariableProfile($ProfileName);
 
             if (($AssociationName == '') && ($AssociationIcon == '')) {
-                throw new \Exception('Removing associations is not implemented yet');
+                unset($keyFound);
+                foreach (self::$profiles[$ProfileName]['Associations'] as $key => $association) {
+                    if ($association['Value'] == $AssociationValue) {
+                        $keyFound = $key;
+                        break;
+                    }
+                }
+                if (isset($keyFound)) {
+                    unset(self::$profiles[$ProfileName]['Associations'][$keyFound]);
+                    usort(self::$profiles[$ProfileName]['Associations'], function ($a, $b)
+                    {
+                        return $a['Value'] - $b['Value'];
+                    });
+                }
```

Comment:
Magst du hier (bei nichtfinden) noch eine Exception werfen mit folgendem Inhalt:

Cannot find association for deletion with value %f

Außerdem fehlt am Ende noch ein "return", damit das Hinzufügen nicht mehr passiert, oder?

Reply:
> Magst du hier (bei nichtfinden) noch eine Exception werfen mit folgendem Inhalt:

Das würde aber nicht dem Produktiven Coding entsprechen. Da passiert dann auch nichts, oder?
Zumindest denke ich, dass ich manchmal auch das Löschen einer nicht existierenden Association mache...

> Außerdem fehlt am Ende noch ein "return", damit das Hinzufügen nicht mehr passiert, oder?

oops, ja.